lib/Runtime/Debug/ProbeContainer.cpp

```diff
@@ -110,7 +110,7 @@ namespace Js
                 ScriptContext* frameScriptContext = walker.GetCurrentScriptContext();
                 Assert(frameScriptContext);
 
-                if (!fMatchWithCurrentScriptContext && !frameScriptContext->IsInDebugMode() && tempFramePointers->Count() == 0)
+                if (!fMatchWithCurrentScriptContext && !frameScriptContext->IsScriptContextInDebugMode() && tempFramePointers->Count() == 0)
```

Comment:
Github does not let me comment for a line 106

will this (isLibraryFrameEnabledDebugger) ever be true? If this gets to true - I doubt that the debugger could handle the library frame in between.
Wondering if this can be removed in the future.